RichMessageKit/Models/Message.swift

```diff
@@ -87,4 +89,6 @@ public struct Message {
 
     /// Content type of the message.
     public var contentType: Message.ContentType
+    
+    public var contactId: String?
```

Comment:
A cell config shouldn't have this extra info unless it's for display or an identifier. We can keep it with other settings.

Reply:
removed